packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,96 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
+
+    outline: none;
+}
+
+.hop-Popover__dialog {
+    --hop-Popover-background: var(--hop-neutral-surface);
+    --hop-Popover-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-font-weight: var(--hop-body-sm-font-weight);
+    --hop-Popover-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-color: var(--hop-neutral-text);
+    --hop-Popover-title-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-title-font-weight: var(--hop-body-sm-semibold-font-weight);
+    --hop-Popover-content-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-content-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-content-color: var(--hop-neutral-text-weak);
+    --hop-Popover-footer-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-footer-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-footer-link-color: var(--hop-primary-text);
+    --hop-Popover-border: 0.0625rem solid var(--hop-neutral-border-weak);
+    --hop-Popover-border-radius: var(--hop-shape-rounded-md);
+    --hop-Popover-box-shadow: var(--hop-elevation-lifted);
+    --hop-Popover-max-inline-width: calc(100% - 1rem);
+    
+    /* Internal variables */
+    --font-family: var(--hop-Popover-font-family);
+    --font-size: var(--hop-Popover-font-size);
+    --font-weight: var(--hop-Popover-font-weight);
+    --line-height: var(--hop-Popover-line-height);
+
+    display: grid;
+    grid-template-areas:
+        "header header"
+        "content content"
+        "footer action";
+    grid-template-columns: repeat(2, auto);
+    gap: var(--hop-space-inset-sm);
```

Comment:
keep in mind that if a footer or header or action isn't present, you will still have a gap for it. You may need some adjustments if you don't always want a gap.

Reply:
here's an example: 
![image](https://github.com/gsoft-inc/wl-hopper/assets/110498164/5730be04-484c-4e1f-b595-702127fdcd1b)